### PR TITLE
Drop daemonizing logic

### PR DESCRIPTION
**Pull request by WhyNotHugo**
Run in foreground, which is what service managers expect. Usages like running `exec swww` in sway's config continue to work fine. User session start-up scripts can simply run `swww &`.

Fixes: https://github.com/LGFae/swww/issues/182

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -14,46 +14,6 @@ use cli::{ResizeStrategy, Swww};
...
-                        socket_path.to_string_lossy()
-                    );
-                    if let Err(e) = std::fs::remove_file(socket_path) {
-                        return Err(format!("failed to delete previous socket: {e}"));
-                    }
-                }
-            }
-            Ok(true) => {
-                return Err("There seems to already be another instance running...".to_string())
-            }
```
Comment (WhyNotHugo) [inline src/main.rs:39]: I think I actually need to move this logic into the daemon, right?

Comment (LGFae) [reply src/main.rs:39]: It's already there, actually.

File: doc/swww-img.1.scd [hunk 2]
```diff
@@ -218,8 +218,7 @@ The images sent will be cached at _$XDG_CACHE_HOME/swww_ or _$HOME/.cache/swww_
 if $XDG_CACHE_HOME does not exist. For each monitor, there will be a file in
 those locations corresponding to the current image/animation being displayed.
 Importantly, **cache will only be loaded during initialization if you use swww
-init**. That is, calling `swww-daemon` directly will **NOT** load the cache, but
-calling `swww-init` will.
+init**. That is, calling `swww-daemon` directly will **NOT** load the cache.
```
Comment (LGFae) [inline doc/swww-img.1.scd:221]: Ah, this isn't true anymore. `swww-daemon` loads the cache now, unless you pass it the `--no-cache`  flag.

File: src/main.rs (hunk 1 above)
Comment (LGFae) [reply src/main.rs:39]: It's already there, actually :).

**Comment by LGFae** [comment]
From what I can tell, this looks done. Are you planning on doing anything else?

**Pull request merged**